### DISCUSSION TITLE
Difference between `spectrum-global-color-` prefixed and normal tokens

### DISCUSSION THREAD

**Discussion by damooo** [adobe/spectrum-web-components#3770] [Q&A] [answered]
Hello, sorry for noob question.
What is thedifference between `spectrum-global-color-gray-500`, and `spectrum-gray-500`? when should we use which?
Thanks.

**Comment by Westbrook** [comment, marked as answer]
Great question. We’ve not officially document it as such, but the `--spectrum-global-` values have been deprecated by Spectrum and will be removed once we’ve completed the work to refactor all of our patterns to no longer use them.

**Reply by damooo** [reply]
Thanks a lot. That clarifies a lot of confusion in understanding tokens.

**Reply by Westbrook** [reply]
Once we finish our internal migration, we'll announce the deprecation and set a transition timeline. So, they're not likely to _disappear_ in the near term, we may even keep them as an opt-in tool after we clarify the removal process, but the sooner applications move off consuming them the easier their future maintenance will be.
